cunumeric/deferred.py

```diff
@@ -787,6 +787,44 @@ def convolve(self, v, lhs, mode):
 
         task.execute()
 
+    @auto_convert([1])
+    def fft(self, lhs, axes, kind, direction):
+        # For now, deferred only supported with GPU, use eager / numpy for CPU
+        if self.runtime.num_gpus == 0:
+            lhs_eager = lhs.runtime.to_eager_array(lhs)
+            self.runtime.to_eager_array(self).fft(
+                lhs_eager, axes, kind, direction
+            )
+            lhs.base = lhs.runtime.to_deferred_array(lhs_eager).base
+        else:
+            input = self.base
+            output = lhs.base
+
+            task = self.context.create_task(CuNumericOpCode.FFT)
+            p_output = task.declare_partition(output)
+            p_input = task.declare_partition(input)
+
+            task.add_output(output, partition=p_output)
+            task.add_input(input, partition=p_input)
+            task.add_scalar_arg(kind.value, ty.int32)
+            task.add_scalar_arg(direction.value, ty.int32)
+            if axes is not None:
```

Comment:
can `axes` ever be `None`? as far as I can tell, that case is already pruned out by the frontend...